### PR TITLE
[REVIEW] Java bindings for clamp [skip ci]

### PR DESCRIPTION
**Pull request by razajafri**
Java bindings for clamp were added to the ColumnVector class.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 ## New Features
 
+- PR #4564 Added Java bindings for signum operator.
```
Comment (harrism) [inline CHANGELOG.md]: Put this closer to the order it was added (i.e. at the end).

**Comment by revans2** [comment]
Why not just expose clamp through JNI and implement signum in java?  This library is supposed to expose the underlying cudf APIs, not necessarily build too much functionality on top of them.

**Comment by razajafri** [comment]
@revans2 good point Bobby, I thought about doing that but then convinced myself that per that logic signum shouldn't even be in cudf/java but should be in the plugin code. Do you think we should expose the clamp only and implement signum operator in the plugin?

File: java/src/main/java/ai/rapids/cudf/ColumnVector.java [hunk 2]
```diff
@@ -1804,6 +1804,108 @@ public ColumnVector stringContains(Scalar compString) {
...
+    return new ColumnVector(clamper(this.getNativeView(), lo.getScalarHandle(),
+        loReplace.getScalarHandle(), hi.getScalarHandle(), hiReplace.getScalarHandle()));
+  }
+
+  /**
+   * Convenience method for integral types for ai.rapids.cudf.ColumnVector#clamp(ai.rapids.cudf.Scalar,
+   *                            ai.rapids.cudf.Scalar, ai.rapids.cudf.Scalar, ai.rapids.cudf.Scalar)
+   *
+   */
+  public ColumnVector clamp(long lo, long loReplace, long hi, long hiReplace) {
```
Comment (revans2) [inline java/src/main/java/ai/rapids/cudf/ColumnVector.java]: Let's just stick with the scalar API until we have a use case that requires something else.  This feels like a lot of code with little benefit.

Comment (razajafri) [reply java/src/main/java/ai/rapids/cudf/ColumnVector.java]: After offline discussion, I have removed the API. PTAL @revans2

**Comment by harrism** [comment]
Please apply  "Ready to merge" label when ready.

**Pull request merged**